content/1.7/back-office/orders/orders/view-shopping-cart.md

```diff
@@ -0,0 +1,36 @@
+---
+title: View Shopping Cart listing
+weight: 1
+---
+
+# SELL > Orders > Shopping Carts > View
+
+[View Shopping Cart](static/img/orders-shopping-carts-view-shopping-cart.png)
+
+This dashboard is displaying the information about the Shopping Cart initiator - the customer, visitor or guest visitor who generated the Cart in Prestashop. It can be accessed by selecting the Shopping Cart row and clicking on View CTA button on the right of the row.
+
+## The elements of the page UI
+
+### Registered customer in Prestashop
+
+- **Total Cart** - the numeric value, that describes the total amount that single customer has so far.
+- **Customer information** - a section, that describes the customer first name, last name, account registration date, valid orders placed, total spent since registration, and the customer email. The customer name and last name contains a link, that directs to the Customer Account page in Prestashop. Email link is also implemented with a link, once clicked initiates the email composing.
```

Comment:
```suggestion
- **Customer information** - a section, that lists the customer:
-  first name,
- last name, 
- account registration date, 
- valid orders placed, 
- total spent since registration, 
- the customer email.
The customer name and last name contains a link, that directs to the Customer Account page in Prestashop. Email link is also implemented with a link, once clicked initiates the email composing.
```

Reply:
We can describe what  "valid orders placed" means

Reply:
updated